docker/resque.yml

```diff
@@ -0,0 +1 @@
+development: redis:6379
```

Comment:
I believe you can use ERB in this file, so how about removing this file and changing `config/resque.yml` to look like:

```yaml
development: <%= ENV.fetch('REDIS_URL', 'localhost:6379') %>
```

And instead of using `volumes` for this override, set `REDIS_URL` in `docker-compose.yml` to be `redis:6379`?